server/src/db/schemas/compliance.js

```diff
@@ -0,0 +1,88 @@
+const mongoose = require('mongoose');
+
+// SCHEMA FORM CONTENT
+const ComplianceDetailSchema = new mongoose.Schema({
+    bbSpecification: {
+        type: String,
+        required: true
+    },
+    bbVersion: {
+        type: String,
+        required: true
+    },
+    status: {
+        type: String,
+        enum: ['in review', 'approved', 'rejected'],
+        default: 'in review'
+    },
+    submissionDate: {
+        type: Date,
+        default: Date.now
+    },
+    deploymentCompliance: {
+        isCompliant: {
+            type: Boolean,
+            required: true
+        },
+        details: {
+            type: String
+        }
+    },
+    requirementSpecificationCompliance: {
+        level: {
+            type: Number,
+            enum: [-1, 1, 2]
```

Comment:
please change literals to meaningful enum class attribute reference. Similar as for status: 
```js 
const SpecificationComplianceLevel = {
  NA: -1,
  LEVEL_1: 1,
  LEVEL_2: 2
};
```
and then in schema: 
```js
  level: {
    type: Number,
    enum: Object.values(SpecificationComplianceLevel),
    required: true
  }
```

Reply:
thanks I will implement it in both "levels"